GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -163,16 +163,9 @@ public FormBrowse([NotNull] GitUICommands commands, string filter, ObjectId sele
             ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
             {
                 await TaskScheduler.Default;
-
-                try
-                {
-                    PluginRegistry.Initialize();
-                }
-                finally
-                {
-                    await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
-                    RegisterPlugins();
-                }
+                PluginRegistry.Initialize();
+                await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+                RegisterPlugins();
```

Comment:
💭 It seems we have a race condition, where the dialog can be closed before plugin registration completes. What are we expecting to happen in this case?

Reply:
`formBrowse` is the main form, if it closed the process is finished, so it probably doesn't matter at this point...